man/systemd-nspawn.xml

```diff
@@ -307,7 +312,11 @@
         <option>--verity-data=</option> (and optionally <option>--root-hash-sig=</option>) options.</para>
 
         <para>Any other partitions, such as foreign partitions or swap partitions are not mounted. May not be specified
-        together with <option>--directory=</option>, <option>--template=</option>.</para></listitem>
+        together with <option>--directory=</option>, <option>--template=</option>.</para>
+
+        <para>In case the name of the parent directory ends with <filename>auto.d</filename> suffix, the path
```

Comment:
please say `auto.d/`, i.e. suffix the `/`, to clarify it's a directory. We try to do this in all our docs: suffix dirs with a slash.

Reply:
fixed